docs/admin/publicaties/index.rst

```diff
@@ -128,6 +130,14 @@ Op een *publicatie*-registratie zijn de volgende metadata beschikbaar. Op het sc
 * ``Geregistreerd op``. De niet-wijzigbare datum en tijd waarop de publicatie nieuw is toegevoegd.
 * ``Laatst gewijzigd op``. De niet-wijzigbare datum en tijd waarop de publicatie voor het laatst gewijzigd was.
 
+**Bewaartermijn**
+
+* ``Bron bewaartermijn``. De naam van de bron van de bewaartermijn.
+* ``Selectiecategorie``. De specifieke categorie binnen de bron van de bewaartermijn.
+* ``Archiefnominatie``. Een selectievakje die aangeeft of de *publicaties* vernietigd of gearchiveerd moet worden.
```

Comment:
"Een selectievakje" is a checkbox. I believe this is currently rendered as a dropdown. Probably the best UX here is a radio widget though, so I'd change that and adapt the documentation accordingly.